### ISSUE TITLE
Call drops when press back or kill the app

### ISSUE THREAD

**Issue by walkoverricha** [SimpleMobileTools/Simple-Dialer#104]
I make a call and kill the app from the background, it drops the call.
Also, when I press back on calling screen, it drops the call

**Comment by tibbi** [comment]
that is currently the expected behaviour, it might change in the future

**Comment by gsantner** [comment]
Is there any chance for this to be changed? It's really trouble some to be in call with doctor, talking about appointment...want to check calendar and boom the call is ended.

Please make phone app & call staying in background, no matter what - until "hang up" is clicked specifically from somewhere.

**Comment by tibbi** [comment]
it is obviously on the roadmap, but it is quite difficult to do. And as the app is still really new, I just went for the simpler solution which works just fine in most cases.

**Comment by gsantner** [comment]
potential idea, maybe something along that works?::

```
override onBackPressed() {
  if (inCall) {
     Intent homeScreen = new Intent(Intent.ACTION_MAIN); 
     homeScreen.addCategory(Intent.CATEGORY_HOME);
     homeScreen.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
     startActivity(homeScreen);
     return true
  }
  return super.onBackPressed
 }
```

**Comment by gsantner** [comment]
Oh, I see it's also about "swiping out" the app in recents list. Just had a call closed in that way. 😃

**Comment by tibbi** [comment]
it is far lot more complicated than this

**Comment by gsantner** [comment]
@tibbi 
Just wanted to give some feedback - I've tried various (open source) dialers and this one is more hasslefree working than the others. Hope you keep working on the project, there is definitely a market hole to fill, there are no other competing alternatives that work well. I.e. when the big core issues are sorted out it may get the best alternative to the google&aosp dialer.

**Comment by HappyClem** [comment]
Hi,
I don't have a back button nor a home button because I use gesture to navigate my phone. There is no way for me to put the dialer app in the background during a call.
Any workaround for now ?

**Comment by seniorm0ment** [comment]
this is a really annoying issue, back should drop you back to the previous screen, and you can end the call via notification (or by clicking back into the call and ending it).
This is how most other apps deal with this.

**Comment by tibbi** [comment]
should be fixed in 5.10.0

**Comment by gsantner** [comment]
thank you @KryptKode for getting this fixed
